packages/react-components/react-tags/src/components/Tag/Tag.types.ts

```diff
@@ -1,17 +1,30 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { Avatar } from '@fluentui/react-avatar';
 
 export type TagSlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<'div'>>;
+  content?: Slot<'span'>;
+  avatar?: Slot<typeof Avatar>;
+  icon?: Slot<'span'>;
+  primaryText?: Slot<'span'>;
+  secondaryText?: Slot<'span'>;
+  dismissButton?: NonNullable<Slot<'button'>>;
```

Comment:
nit: since the PR is mainly boiler plate stuff, would you mind adding descriptions to the slots and props?